psycopg/connection_type.c

```diff
@@ -60,6 +60,8 @@ psyco_conn_cursor(connectionObject *self, PyObject *args, PyObject *kwargs)
 {
     PyObject *obj = NULL;
     PyObject *rv = NULL;
+    PyObject *close;
```

Comment:
I would prefer to use a `{block}` in the `error:` handler and the fine the variables in that scope.